launcher/config.go

```diff
@@ -103,6 +103,19 @@ func WithSpanExporterInsecure(insecure bool) Option {
 	}
 }
 
+// WithSpanSamplingPercent sets a percentage of spans to be sampled client-side.
+// For instance, if set to 75, then 25% of spans will be dropped and 75% will be
+// kept. The default value is 100 (no sampling).
+//
+// Sampling is applied at the trace level, rather than the span level, so either
+// all the spans in a trace are sampled, or none are.
+func WithSpanSamplingPercent(percent int) Option {
```

Comment:
:thinking: this interface means that if span sampling is enabled in the environment, code can't override it.

I guess there should be a separate `WithSpanSamplingEnabled()` option? Or we could have a single `WithSpanSampling(percent int, enabled bool)`?

Reply:
I've added a separate `WithSpanSamplingEnabled()` for now, but happy to talk about it more.